.github/workflows/elixir.yml

```diff
@@ -0,0 +1,119 @@
+name: Elixir CI
+
+on:
+  pull_request:
+  push:
+    branches: main
+
+concurrency:
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
This is very cool! I updated my gtfs_creator workflow to use it, and just pushed up a squashed commit. So neat to see it cancel the one that was going and run my newest one rather than just keeping it queued.

Reply:
@bklebe can you also port this back to the default template in mbta/.github ?

Reply:
will do